### PR TITLE
Created a branch of bodyviz fork as submodule

### PR DESCRIPTION
**Pull request by kvijaykiran**
Created a branch of bodyviz fork as submodule. Added assembly definitions:
- UnityVolumeRendering.asmdef in EasyVolumeRendering >> Assets
- UnityVolumeRendering.Editor.asmdef in EasyVolumeRendering >> Assets >> Editor

I made these changes within bodyviz_unity project's submodule, and then pushed the changes.

### CODE REVIEW

File: Assets/Scripts/Importing/ImageSequenceImporter/SimpleITK/SimpleITKDICOMImporter.cs.meta [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+guid: c7176da21a65a4fbbab548f6227ab34a
+MonoImporter:
+  externalObjects: {}
+  serializedVersion: 2
+  defaultReferences: []
+  executionOrder: 0
+  icon: {instanceID: 0}
+  userData: 
+  assetBundleName: 
+  assetBundleVariant:
```
Comment (jholub15) [inline Assets/Scripts/Importing/ImageSequenceImporter/SimpleITK/SimpleITKDICOMImporter.cs.meta:11]: What is this file being used for? It looks like something the ITK installation might have left behind so I'm asking to make sure there is no conflict in the future.

**Review by jholub15 — COMMENTED** [review]
There are just a ton of meta files being added here. I'm not sure if it is worth going through all of them or not. It gives me pause when some of them are for C# files that don't seem to have changes. Is there a reason these meta files are changing? 

Is there some way for me to test this?

It seems like you are adding a couple assembly definitions files, so to have all these other meta files change seems weird. With all of these changing, I'm just envisioning the next pull we do from the developer is going to have 140-ish conflicts at the minimum.

File: Assets/Scripts/Importing/ImageSequenceImporter/SimpleITK/SimpleITKDICOMImporter.cs.meta (hunk 1 above)
Comment (kvijaykiran) [reply Assets/Scripts/Importing/ImageSequenceImporter/SimpleITK/SimpleITKDICOMImporter.cs.meta:11]: .meta files are automatically created, one for every file and a folder in a Unity project. They store some essential info about the actual files they belong to. Unfortunately, we'll have to track them in git and cannot get rid of them.

**Comment by kvijaykiran** [comment]
> There are just a ton of meta files being added here. I'm not sure if it is worth going through all of them or not. It gives me pause when some of them are for C# files that don't seem to have changes. Is there a reason these meta files are changing?

No idea.
> 
> Is there some way for me to test this?
Don't know.

> 
> It seems like you are adding a couple assembly definitions files, so to have all these other meta files change seems weird. With all of these changing, I'm just envisioning the next pull we do from the developer is going to have 140-ish conflicts at the minimum.
Internally, I do not know what happens when assembly files are created. From your comment, I'm assuming that any dependencies that an assembly file will have might have its meta file content changed. I'd say just ignore any meta files if they show up in diff.

**Pull request merged**